### ISSUE TITLE
Fields defined in inline type are not shown

### ISSUE THREAD

**Issue by zyguan** [mulesoft/api-console#526]
## Expected outcome

Hi, I found that fields defined in inline type are not shown since v3.0.11.

## Actual outcome

![](https://cdn-business.discourse.org/uploads/raml/original/1X/98ea308fc27995548ce8a79afc6340bd088c1199.png)

## Steps to reproduce

http://forums.raml.org/t/inline-type-inheritance/1763/8

```yaml
#%RAML 1.0
title: API with Types
types:
  User:
    type: object
    properties:
      firstname: string
      lastname:  string
      age:       number
/users/{id}:
  get:
    responses:
      200:
        body:
          application/json:
            type: User
            properties:
              token: string  # this field is missing
```

## Was this working before?

- [x] Yes, it was working before
- [ ] No, it was already broken
- [ ] I don't know / not sure

## Affected browsers

- [x] All or I don't know
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer <\3
- [ ] Edge
- [ ] Safari
- [ ] Other (please, specify)

**Comment by zyguan** [comment]
BTW, I use the inline type for some boilerplate response.  eg. one of my project defines all reponses like `{ "status": 907, "data": ... }`, so, I defined a base type
```yaml
Response:
  type: object
  properites:
    status: integer
```
and for every response, the body was defined like
```yaml
200:
  body:
    type: Response
    properties:
      data: User
```

I'm new to RAML, is it the right way to do things like that?

**Comment by carowright** [comment]
Hi @zyguan!
Thank you for reporting. This fix will be available in next version (3.0.27)

**Comment by zyguan** [comment]
Great! I look forward to the next version.
